gen/declarative.tmpl

```diff
@@ -0,0 +1,77 @@
+{{ template "header" . }}
+on:
+  push:
+    branches: [ {{ .DefaultBranch }} ]
+  pull_request:
+    branches: [ {{ .DefaultBranch }} ]
+  workflow_dispatch:
+{{- template "permissions" . }}
+jobs:
+  plan:
+    if: {{`${{ github.event_name == 'pull_request' }}`}}
+    {{- template "job-setup" . }}
+      - uses: ariga/atlas-action/schema/plan@master
+        with:
+          {{- if .From }}
+          from: {{ .From }}
+          {{- end }}
+          {{- if .CreateDevURL }}
+          {{ template "UseServices" . }}
+          {{- end -}}
+          {{- with .ConfigPath }}
+          config: 'file://{{ . }}'
+          {{- end }}
+          {{- with .Env }}
+          env: '{{ . }}'
+          {{- end }}
+        env:
+          GITHUB_TOKEN: {{`${{ github.token }}`}}
+  apply:
+    if: {{`${{ github.event_name == 'push' && github.ref == 'refs/heads/`}}{{ .DefaultBranch }}{{`' }}`}}
+    {{- template "job-setup" . }}
+      - uses: ariga/atlas-action/schema/push@master
+        with:
+          {{- if .CreateDevURL }}
+          {{ template "UseServices" . }}
+          {{- end -}}
```

Comment:
Template name should be lower case: `use-service`

But, I think this template should be a Go function that print URL depends on Driver.

Reply:
I'll make it a func